README.md

```diff
@@ -58,6 +59,7 @@ paint_mode=brush
 - `text_size` is the default text size (must be between 10 and 50)
 - `text_font` is the font used to render text, its format is pango friendly
 - `paint_mode` is the mode activated at application start (must be one of: brush|text|rectangle|ellipse|arrow|blur, matching is case-insensitive)
+- `quit_after_export` is used to make the application exit after saving the picture or copying it to the clipboard 
```

Comment:
`early_exit` here and the corresponding codebase.

Reply:
early_exit is indeed a lot better as a name, it has been changed now.